docs/arf.md

```diff
@@ -551,43 +561,43 @@ mentioned depending on the specific role and their criticality for
 example the different roles and actors involved with remote signing
 processes.
 
-When used, Trusted List[^9] need to provide a registration service for
+When used, Trusted List[^11] need to provide a registration service for
 the relevant entities, maintain a registry and enable third party access
 to the registry information. The terms and conditions of entities to
 become registered are for each registrar to determine unless specified
-in e.g., sectoral rules.
+in e.g., EU law or sectoral rules.
 
-#### 4.1.5. Qualified Electronic Attestation of Attributes Providers
+#### 4.1.5 Qualified Electronic Attestation of Attributes Providers
 
-Qualified EAA are provided by QTSPs. The general Trust Framework for
+Qualified EAA are provided by QTSPs. The IDAS Regulation Trust Framework for
```

Comment:
as per @duviner change `IDAS` to `eIDAS`

Reply:
Fixed in https://github.com/skounis/architecture-and-reference-framework/commit/80d00cf5ad1c3930235e4140b1fc8a975638f787